ETL/ExcelParsing/ExcelParser.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using OfficeOpenXml;
+using System.IO;
+using ETL.ExcelParsing.DTO;
+
+namespace ETL.ExcelParsing
+{
+  public class ExcelParser
```

Comment:
This might actually be two classes.  One that can take commands to read from excel and one that has knowledge of our exact Excel structure.